source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -374,7 +374,10 @@ void Filter::continueDecoding() {
 }
 
 Filter::PerRouteFlags Filter::getPerRouteFlags(const Router::RouteConstSharedPtr& route) const {
-  if (route == nullptr || route->routeEntry() == nullptr) {
+  if (route == nullptr ||
+      (!Runtime::runtimeFeatureEnabled(
```

Comment:
Wouldn't this affect redirects too? Not sure if these be passed to ext_authz?

Reply:
good point, I tested the redirect case, this change affect it.

@mattklein123 WDYT?

Reply:
Yeah I think it's correct to fix redirect also. You might want to rename the runtime flag and update the release notes?

/wait

Reply:
right, let me update